format/proto/types.proto

```diff
@@ -20,14 +20,16 @@ enum OrchestratorRequestType {
     QUERY = 0;
     // Add a new node to the cluster
     ADD_NODE = 1;
+    // Remove node from the cluster
+    REMOVE_NODE = 2;
 }
 
 // Orchestrator responses
 enum OrchestratorResponseType {
     // The status of a workflow
     WORKFLOW_STATUS = 0;
     // Id of a created workflow
-    WORKFLOW_ID = 1;
+    WORKFLOW_CREATE = 1;
```

Comment:
What is workflow_create? The comments don't match...

Reply:
Changed the name.